ci-operator/jobs/openshift/ci-secret-mirroring-controller/openshift-ci-secret-mirroring-controller-master-presubmits.yaml

```diff
@@ -4,6 +4,7 @@ presubmits:
     always_run: true
     branches:
     - master
+    cluster: ci/api-build01-ci-devcluster-openshift-com:6443/system:serviceaccount:ci:config-updater
```

Comment:
I think the generator is not going to like this -- can we make a copy of the job?

Reply:
On it.

Reply:
Job copied.
How can we deal with the generator failure?

Reply:
Never mind ... i should have known the answer.